bayesian/default_config.py

```diff
@@ -53,3 +53,5 @@
 GEMINI_SERVER_URL = "http://{host}:{port}".format(
                      host=os.getenv('F8A_GEMINI_SERVER_SERVICE_HOST'),
                      port=os.getenv('F8A_GEMINI_SERVER_SERVICE_PORT'))
+
+COMPONENT_ANALYSES_LIMIT = 10
```

Comment:
Can we have this set via OS template? This way we can dynamically change the limits via saas-analytics PR